manifests/bosh-manifest/deployments/010-bosh-template.yml

```diff
@@ -81,7 +85,7 @@ jobs:
     ntp: (( grab meta.ntp ))
 
     agent:
-      mbus: (( concat "nats://nats:" secrets.bosh_nats_password "@" meta.bosh_service_ip ":4222" ))
```

Comment:
It was used but I replaced it with `terraform_outputs.bosh_fqdn` as we want to switch to hostname.